data/brands/shop/car_repair.json

```diff
@@ -25,6 +25,16 @@
     }
   },
   "items": [
+    	{
+	      "displayName": "Eneos Frontier",
+	      "locationSet": {"include": ["jp"]},
```

Comment:
If this is a Japan-only entry, I would recommend following https://wiki.openstreetmap.org/wiki/JA:Naming_sample to complete other multilingual tags. Of course, this is not mandatory, but it can reduce the workload of other maintainers who mainly maintain Japan-related entries.